source/extensions/common/dynamic_forward_proxy/dns_cache_impl.cc

```diff
@@ -242,7 +244,11 @@ DnsCacheImpl::PrimaryHostInfo* DnsCacheImpl::createHost(const std::string& host,
                          *this, std::string(host_attributes.host_),
                          host_attributes.port_.value_or(default_port),
                          host_attributes.is_ip_address_, [this, host]() { onReResolveAlarm(host); },
-                         [this, host]() { onResolveTimeout(host); }))
+                         [this, host]() {
+                           if (enable_dfp_resolve_timeout_) {
```

Comment:
looks like this should be dns_cache_resolve_timeout?

Reply:
Outdated comment.